nginx-sys/src/lib.rs

```diff
@@ -70,11 +70,9 @@ pub use bindings::*;
 /// let data: &str = "example"; // The string to convert
 /// let ptr = str_to_uchar(pool, data);
 /// ```
-pub fn str_to_uchar(pool: *mut ngx_pool_t, data: &str) -> *mut u_char {
+pub unsafe fn str_to_uchar(pool: *mut ngx_pool_t, data: &str) -> *mut u_char {
```

Comment:
why is that? when unsafe is used in the body of the fn, it is `safe` to invoke it, otherwise, the caller must use `unsafe` to `unsafe { str_to_uchar(...) }` . it is definitely might break many existing codes, let's keep it back. 
what's the reason?